docs/chapter-6.mdx

```diff
@@ -89,6 +89,8 @@ export function reducer(state: AppState, action: Action) {
 }
 ```
 
+> **_NOTE:_** Beware of the **payload** attribute of the Action, which only fits the "old" way to create actions. Next example will use the props and call directly the properties added to the Action.
```

Comment:
Not related to this PR, but what's the benefit of writing reducers the old way? 
I'm afraid that this could confuse readers.

Reply:
Good question, I also wander. 
Maybe did Lars wrote some of the content a while ago ?
If it was about me, I would only write the "create" way.

Reply:
This course is the impressive work of @Armenvardanyan95. Thoughts on this, Armen?

Reply:
Armen is busy so we will ignore Tim's comment for now.